envs/sandbox/main.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  # UPDATE: set this to your stack (dev,prod,imp, etc)
+  stack = "dev" # TODO: update this to "sandbox" for clarity
```

Comment:
should this be done?

Reply:
Eventually this should be updated to "sandbox," however the actual live vpc is currently named `circleci-dev`, so we'll need to rename the vpc before we can update this value, otherwise the data call in [modules/cms/main.tf](https://github.com/CMSgov/enterprise-setup/blob/master/modules/cms/main.tf#L3) will fail since it would be looking for a nonexistent vpc named `circleci-sandbox`.